home03/form.py

```diff
@@ -0,0 +1,12 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, PasswordField, DateField, BooleanField
+from wtforms.validators import DataRequired, Email, Length, EqualTo
+
+
+class RegisterForm(FlaskForm):
+    user_name = StringField('Имя', validators=[DataRequired()])
+    user_email = StringField('Email', validators=[DataRequired(), Email()])
```

Comment:
Тут можно использовать `EmailField`
```suggestion
    user_email = EmailField('Email', validators=[DataRequired(), Email()])
```